src/main/java/au/org/emii/geoserver/extensions/filters/layer/data/Filter.java

```diff
@@ -8,7 +8,7 @@
 package au.org.emii.geoserver.extensions.filters.layer.data;
 
 import org.apache.commons.lang.StringUtils;
-
+import java.util.*;
```

Comment:
I'm not a java expert, but the best practice is to import just what you need I think.